packages/frontend/.eslintrc.js

```diff
@@ -53,6 +54,9 @@ module.exports = {
             'message': 'Please use a selector for any state accesses within useSelector'
         }]
     },
+    settings: {
+        'import/ignore': ['src/config/*'],
```

Comment:
`config.js` produced lots of false positives, I'm assuming because individual keys aren't exported directly but do get destructured in our `import`s. I didn't see a way to get around this besides whitelisting the module but if there's a better way to do this I think it's worth checking out.